### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by rishiwardhan**
hi, just added new lines to the project

### CODE REVIEW

**Comment by rishiwardhan** [comment]
hi, just added new lines to the projects to learners to understand even more better

**Comment by rishiwardhan** [comment]
please check the lines and merge it .

File: readme.md [hunk 1]
```diff
@@ -292,28 +295,30 @@ Modern electronic component distributors often further sub-categorize integrated
...
 ### Functions of an OS
-- **Convenience**: An OS makes a computer more convenient to use.
-- **Efficiency**: An OS allows the computer system resources to be used efficiently.
+- **Convenience  of os**: An OS makes a computer more convenient to use.
+- **Efficiency of os**: An OS allows the computer system resources to be used efficiently.
 - **Ability to Evolve**: An OS should be constructed in such a way as to permit the effective development, testing, and introduction of new system functions at the same time without interfering with service.
 - **Throughput**: An OS should be constructed so that It can give maximum throughput(Number of tasks per unit time).
 
-### Major functionalities of an OS
+### Major functionalities of an OS (operating system)
```
Comment (shhossain) [inline readme.md:308]: Remove this

File: readme.md [hunk 2]
```diff
@@ -292,28 +295,30 @@ Modern electronic component distributors often further sub-categorize integrated
...
 - **Memory Management**: Refers to the management of primary memory. The operating system has to keep track, how much memory has been used and by whom. It has to decide which process needs memory space and how much. OS also has to allocate and deallocate the memory space.
 - **Security/Privacy Management**: Privacy is also provided by the Operating system by means of passwords so that unauthorized applications can’t access programs or data. For example, Windows uses **_Kerberos_** authentication to prevent unauthorized access to data.
 
+"Kerberos" is a computer-network authentication protocol that works on the basis of tickets to allow nodes communicating over a non-secure network to prove their identity to one another in a secure manner
+
 ## [Memory and Storage](Memory%20and%20Storage/readme.md)
-If you are unsure about the difference between memory and storage in computers, this article covers the differences between the two.
+If you are not aware of the difference between memory and storage in computers, this article covers the differences between the two. {memory and storage}
```
Comment (shhossain) [inline readme.md:318]: Don't need to change

File: readme.md [hunk 3]
```diff
@@ -292,28 +295,30 @@ Modern electronic component distributors often further sub-categorize integrated
...
 
 ### Memory
-The term _memory_ refers to the component within your computer that allows for short-term data access. You may recognise this component as DRAM, or dynamic random-access memory. Your computer performs many operations by accessing data stored in its short-term memory. Some examples of such operations include editing a document, loading applications and browsing the internet. The speed and performance of your system depends on the amount of memory that is installed on your computer.
+The term "memory" refers to the component within your computer that allows for short-term data access. You may recognise this component as DRAM, or dynamic random-access memory. Your computer performs many operations by accessing data stored in its short-term memory. Some examples of such operations include editing a document, loading applications and browsing the internet. The speed and performance of your system depends on the amount of memory that is installed on your computer.
```
Comment (shhossain) [inline readme.md:321]: Italic looks better

File: readme.md [hunk 4]
```diff
@@ -327,7 +332,7 @@ While the desk represents the computer's memory, the filing cabinet represents t
...
 
-There are many kinds of file systems, each with unique structure and logic, properties of speed, flexibility, security, size and more. Some file systems have been designed to be used for specific applications. For example, the ISO 9660 file system is designed specifically for optical discs.
+There are many kinds of file systems, each has unique structures and logic, properties of speed, flexibility, security, size and more. Some file systems have been designed to be used for specific applications. For example, the ISO 9660 file system is designed specifically for optical discs.
```
Comment (shhossain) [inline readme.md:335]: Not needed

File: readme.md [hunk 5]
```diff
@@ -292,28 +295,30 @@ Modern electronic component distributors often further sub-categorize integrated
...
+An operating system (often abbreviated to Os) acts as an intermediary between the user of a computer and computer hardware. The purpose of an operating system is to provide an environment in which an user can execute programs conveniently and efficiently.
 An operating system is a software that manages computer hardware. The hardware must provide appropriate mechanisms to ensure the correct operation of the computer system and to prevent user programs from interfering with the proper operation of the system.
 An even more common definition is that the operating system is the one program running at all times on the computer (usually called the kernel), with all else being application programs.
 
 ### Functions of an OS
-- **Convenience**: An OS makes a computer more convenient to use.
-- **Efficiency**: An OS allows the computer system resources to be used efficiently.
+- **Convenience  of os**: An OS makes a computer more convenient to use.
```
Comment (shhossain) [inline readme.md:303]: Simple is better

File: readme.md [hunk 6]
```diff
@@ -292,28 +295,30 @@ Modern electronic component distributors often further sub-categorize integrated
...
+- **Efficiency of os**: An OS allows the computer system resources to be used efficiently.
```
Comment (shhossain) [inline readme.md:304]: Simple is better. Remove this

File: readme.md [hunk 7]
```diff
@@ -292,28 +295,30 @@ Modern electronic component distributors often further sub-categorize integrated
 - Three-dimensional integrated circuits (3D ICs) are categorized into through-silicon via (TSV) ICs and Cu-Cu connection ICs.
 
 ## [Operating Systems](Operating%20Systems/readme.md)
-An operating system (or OS for short) acts as an intermediary between the user of a computer and computer hardware. The purpose of an operating system is to provide an environment in which an user can execute programs conveniently and efficiently.
+An operating system (often abbreviated to Os) acts as an intermediary between the user of a computer and computer hardware. The purpose of an operating system is to provide an environment in which an user can execute programs conveniently and efficiently.
```
Comment (shhossain) [inline readme.md:298]: Unnecessary

**Review by shhossain — CHANGES_REQUESTED** [review]
Please remove the commented lines.

**Comment by shhossain** [comment]
@rishiwardhan you've added lots of unnecessary words or changed to similar words. The issue is about adding something new or fixing typos if spotted. Try to add new things next time.

**Comment by rishiwardhan** [comment]
Oh , ok then can I add new materials and fix typos

**Comment by shhossain** [comment]
@rishiwardhan yes you can.

**Comment by rishiwardhan** [comment]
Ok thank you

**Comment by rishiwardhan** [comment]
Hi, yesterday I have added about Kerberos

**Pull request closed without merging**